docs/platforms/javascript/common/tracing/configure-sampling/index.mdx

```diff
@@ -11,15 +11,15 @@ Sentry's tracing functionality helps you monitor application performance by capt
 The JavaScript SDK provides two main options for controlling the sampling rate:
 
 1. Uniform Sample Rate (`tracesSampleRate`)
-This option sets a fixed percentage of transactions to be captured:
+`tracesSampleRate` is floating point value 0.0 and 1.0, which controls the probability that a transaction will be sampled.
 
 <PlatformContent includePath="/tracing/sample-rate" />
 
-With `tracesSampleRate` set to `0.25`, approximately 25% of transactions will be recorded and sent to Sentry. This provides an even cross-section of transactions regardless of where in your app they occur.
+With `tracesSampleRate` set to `0.25`, each transaction in your application is randomly sampled with a probability of `0.25`, so you can expect that one in every four transactions will be sent to Sentry.
```

Comment:
```suggestion
With `tracesSampleRate` set to `0.25`, each transaction in your application is randomly sampled with a probability of 25%, so you can expect that one in every four transactions will be sent to Sentry.
```

maybe a bit easier to understand?

Reply:
So originally i had gone with percentage also, by in the Python PR that @sfanahata was working on, @szokeasaurusrex mentioned that a percentage was wrong to use. I def agree that percentage translates better to users.

Reply:
25% is not wrong to use, because a sample rate of 0.25 means that there is a 25% probability of sampling each transaction. 

My comment on the Python PR was referring to calling the traces sample rate the "percentage" of transactions which will be captured. Calling this rate a percentage is wrong, because it is specified as a fractional value between zero and one, not as a percentage. E.g. it is invalid to set the sample rate to 25; if you want to capture 25% of transactions you have to set the rate to 0.25. 

Sorry for the confusion. Does the distinction make sense now?